pipelines/controller/config/application.yaml

```diff
@@ -69,7 +71,7 @@ fhirdata:
   # Equivalent to pipeline `resourceList` parameter.
   # Note there is no Questionnaire in our test FHIR server, but it is okay; see
   # https://github.com/google/fhir-data-pipes/issues/785.
-  resourceList: "Patient,Encounter,Observation,Questionnaire"
+  resourceList: "Patient,Encounter,Observation,Questionnaire,Condition,Practitioner,Location,Organization"
```

Comment:
To be consistent, can we have this change at `docker/config/application.yaml` as well?

Reply:
Resolving this as it is related to first PR.